tools/java/org/unicode/cldr/tool/SearchCLDR.java

```diff
@@ -364,16 +365,12 @@ public static void main(String[] args) {
         File[] newDirs = new File[sourceDirs.length];
         int item = 0;
         for (File s : sourceDirs) {
-            try {
-                String path = s.getCanonicalPath();
-                int baseLoc = path.lastIndexOf("/cldr/");
-                if (baseLoc < 0) {
-                    throw new ICUUncheckedIOException("source doesn't contain /cldr/");
-                }
-                newDirs[item++] = new File(base, path.substring(baseLoc + 5));
-            } catch (IOException e) {
```

Comment:
No IOException possible using new utilities file.
Here and a few other places in this PR.